DevIncubator_Autopark/Program.cs

```diff
@@ -1,62 +1,30 @@
 ﻿using DevIncubator_Autopark.Engines;
+using DevIncubator_Autopark.Types;
+using DevIncubator_Autopark.VehicleEntity;
 using System;
+using System.IO;
 
 namespace DevIncubator_Autopark
 {
 	public class Program
 	{
+		private static readonly string DirectoryPath = @$"{Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..\\..\\..\\"))}\Files\";
```

Comment:
Just imagine this project will go into production. This line will not work because production app will have compiled code only. So set for your csv file copy on compile and use base directory only.